DevelopGitHub/GitHubExtension/GitHubExtension.DB/PullRequestTemplates.sql

```diff
@@ -0,0 +1,6 @@
+﻿CREATE TABLE [dbo].[PullRequestTemplates]
```

Comment:
Yep, I think it's better idea and add a new field something like IsTemplate with boolean type.